models/surgical_tool_segmentation/docs/readme.md

```diff
@@ -0,0 +1,83 @@
+# Description
+A pre-trained model for the surgical tool segmentation task.
+
+# Model Overview
+This model is trained using a flexible unet structure with an efficient-b0 [1] as the backbone and a UNet architecture [2] as the decoder. Datasets use private samples from Activ-Surgical.
```

Comment:
Let's have a consistent reference to `Activ Surgical` and remove the `-`. Perhaps also a link to their [website](https://www.activsurgical.com/)?

Reply:
Hi @jin-nvidia ,
     Thank you for your advice. I have updated the reference to Activ Surgical and added their webseite link to it.
Thanks,
Bin